odata-mock/package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "compile": "tsc && cd src && copyfiles test/services.xml ../dist/ && copyfiles test/varkes_config.json ../dist/ && copyfiles server/resources/* ../dist/",
```

Comment:
You said once something about a "prestart" goal, why you are not using it instead of compile?

Reply:
`lerna run test` runs tests in parallel and before starting running tests, I need a prestep that compiles the package so that example packages can use it.